server/middleware/populateCards.ts

```diff
@@ -7,7 +7,7 @@ export default function populateCards(locationsService: LocationsService) {
   return asyncMiddleware((req, res, next) => {
     setCanAccess(locationsService)
     res.locals.resiCards = [
-      req.featureFlags.createAndCertify
```

Comment:
I believe that this should be "Manage locations" for all users when the prison has create active, regardless of permissions

Reply:
Even though as a view they can't "Manage locations?"  - happy however if that is true

Reply:
I'll see if I can merge the PRs